src/main/java/RacingCar/RandomMovingStrategy.java

```diff
@@ -0,0 +1,20 @@
+package RacingCar;
+
+import java.util.Random;
+
+public class RandomMovingStrategy implements MoveStrategy {
+    private static final int RANDOM_NUMBER_UPPER_BOUND = 10;
+    private static final int MOVABLE_NUM = 4;
+
+    @Override
+    public boolean isMove() {
+        if (getRandomNumber() >= MOVABLE_NUM) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return getRandomNumber() >= MOVABLE_NUM;
```
이렇게 하면 코드량도 줄어들고 심플해질거 같은데요?😄